### PR TITLE
Feature/back to top component

### PR DESCRIPTION
**Pull request by ReenaOmana**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class ='backtotop-cmp'>
+<div id="back2Top" class = 'bottom-to-top-container'>
+<a class ='bottom-top-icon' 'title="Back to top" href="#"><i class="fa fa-angle-up" aria-hidden="true"></i></a><span>${properties.title}</span>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html]: @ReenaOmana : why is tere a single quote ' before title attribute ?

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/.content.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/.content.xml:1]: @ReenaOmana : footer container component policy has not been updated to add Back to top component

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/clientlibs/less/backtotop.less [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.bottom-to-top-container {
+    justify-content: center;
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/clientlibs/less/backtotop.less:2]: @ReenaOmana : component is not coming in mobile

image

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div class ='backtotop-cmp'>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html:1]: @ReenaOmana : Please add sample text/image for edit mode when component is added

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@ReenaOmana : pl incorporate chnages

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html (hunk 1 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html]: This has been removed

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/.content.xml (hunk 2 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/.content.xml:1]: Footer container policy is updated now

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/clientlibs/less/backtotop.less (hunk 3 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/clientlibs/less/backtotop.less:2]: I'm able to see the component in mobile view Rishabh. Please see the screenshot
mobileview

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html (hunk 4 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html:1]: Have added the text in edit mode as per the comment

**Review by rishabh-behl — APPROVED** [review]
Looks fine , although still not able see bac to top in mobile on my system

**Pull request merged**